package.json

```diff
@@ -20,6 +20,8 @@
     "cannon": "^0.6.2",
     "react": "~15.3.1",
     "react-dom": "^15.3.1",
+    "react-router": "^3.0.0-alpha.3",
```

Comment:
using 3.0 instead of 2.* because of https://github.com/reactjs/react-router/issues/3611#issuecomment-240950257